presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/MergeIdenticalWindows.java

```diff
@@ -142,7 +137,7 @@ private static WindowNode collapseWindows(PlanNode source, SpecificationAndFrame
             SpecificationAndFrame(WindowNode node)
```

Comment:
The optimizer is a lot uglier if you don't use SpecificationAndFrame. I have a commit with that on it, but the code won't win any awards for prettyness :-)